### PR TITLE
Add role variable for cronjob frequency

### PR DESCRIPTION
**Pull request by ClementJ35**
## 🗣 Description ##

This PR add cronjob frequency choice as variable. It defaults to weekly, as it is now.

## 💭 Motivation and context ##

Force weekly scan may be a bit low or high depending of the context

## 🧪 Testing ##

It is tested on CentOS7 and Debian 11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +16,11 @@ None.
...
-None.
 
-<!--
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| optional_variable | Describe its purpose. | `default_value` | No |
-| required_variable | Describe its purpose. | n/a | Yes |
--->
+| clamav_cron_frequency | Set the cronjob frequency of clamscan. Must be one of : `hourly`,`daily`,`weekly` or `monthly` | `weelky` | No |
+
```
Comment (jsf9k) [inline README.md]: Fix typo and tweak language:
```suggestion
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`,`daily`,`weekly`, or `monthly`. | `weekly` | No |
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
Requesting a few very minor changes.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md:23]: Line 23 above is an extraneous blank line, and I think that is what is making the linters fail.  Can you remove it as well?

File: README.md [hunk 2]
```diff
@@ -16,14 +16,9 @@ None.
...
 
-None.
-
-<!--
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| optional_variable | Describe its purpose. | `default_value` | No |
-| required_variable | Describe its purpose. | n/a | Yes |
--->
+| clamav_cron_frequency | Set the cronjob frequency of clamscan. Must be one of : `hourly`,`daily`,`weekly` or `monthly` | `weelky` | No |
```
Comment (jsf9k) [inline README.md]: Fix typo and tweak language:
```suggestion
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
Requesting a few minor changes.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: Outdated.  See [here](https://github.com/cisagov/ansible-role-clamav/pull/37#pullrequestreview-861365215) instead.

**Comment by jsf9k** [comment]
@cisagov/team-ois - We should consolidate the first three commits via a rebase before merging.  I'm not doing it now because I don't want to mess up @ClementJ35's checkout.

File: README.md (hunk 2 above)
Comment (mcdonnnj) [reply README.md]: @jsf9k Would you tweak your suggestion to add spaces after the other commas in the list?

Comment (jsf9k) [reply README.md]: Thanks to @mcdonnnj and his eagle eyes!

**Review by jsf9k — APPROVED** [review]

File: defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+clamav_cron_frequency: "weekly"
```
Comment (dav3r) [inline defaults/main.yml]: We typically prefer to leave the quotes off unless they are necessary.
```suggestion
clamav_cron_frequency: weekly
```

**Review by mcdonnnj — APPROVED** [review]
A nice enhancement 👍

**Review by dav3r — APPROVED** [review]
This looks great to me.  Thanks @ClementJ35 for your contribution here! 🚀

**Pull request merged**